pyfstat/helper_functions.py

```diff
@@ -724,19 +724,12 @@ def get_predict_fstat_parameters_from_dict(signal_parameters, return_lal_keys=Fa
     ----------
     signal_parameters: dict
         Dictionary containing at least the signal parameters required by
-        helper_functions.predict_fstat. This dictionary's keys must follow
-        the PyFstat convention (F0 instead of Freq).
-    return_lal_keys: bool
-        If True, the returned dictionary will contain keys according to the
-        lal convention (Freq instead of F0).
+        helper_functions.predict_fstat.
```

Comment:
```suggestion
        helper_functions.predict_fstat. This dictionary's keys must follow
        the PyFstat convention (F0 instead of Freq).
```

Let's keep these two lines for clarity, even if it's just "yes, global policy isn't violated here" - due to the close connection of the function with a lalapps code, it's worth saying explicitly.